src/xcode/ENA/ENA/Source/Scenes/FriendsInvite/FriendsInviteController.swift

```diff
@@ -40,15 +40,6 @@ final class FriendsInviteController: UIViewController, UIActivityItemSource {
 		descriptionLabel.text = AppStrings.InviteFriends.description
 
 		inviteButton.setTitle(AppStrings.InviteFriends.submit, for: .normal)
-		inviteButton.titleEdgeInsets = UIEdgeInsets(top: 10, left: 10, bottom: 10, right: 10)
-
-		guard let titleLabel = inviteButton.titleLabel else { return }
-		titleLabel.font = UIFontMetrics.default.scaledFont(for: UIFont.systemFont(ofSize: 17, weight: .semibold))
-		titleLabel.adjustsFontForContentSizeCategory = true
-		titleLabel.lineBreakMode = .byWordWrapping
-
-		guard let inviteButton = inviteButton else { return }
-		inviteButton.addConstraint(NSLayoutConstraint(item: inviteButton, attribute: .height, relatedBy: .equal, toItem: inviteButton.titleLabel, attribute: .height, multiplier: 1, constant: 0))
```

Comment:
Similar to ```ResetViewController```, remove these two lines here is breaking the resizing after accessibility class changes. They should be put back.